lib/incito-browser/incito.ts

```diff
@@ -788,13 +793,60 @@ export default class Incito extends MicroEvent<{
 
     start() {
         this.el.addEventListener('click', (e) => {
-            const el = closest(
+            const linkEl = closest(
+                e.target as HTMLElement,
+                '.tjek-incito__view [data-link]'
+            );
+            const carouselEl = closest(
+                e.target as HTMLElement,
```

Comment:
not really related to this PR since this pattern was here before, but you can add `if (!(e.target instanceof HTMLElement`)) return;` at the top of the callback and all of these will be inferred correctly, saving all the casting.

Reply:
Good to know, fixed.